pyspark/bigdl/ml_piepeline/dl_classifier.py

```diff
@@ -0,0 +1,118 @@
+from pyspark.ml.pipeline import Estimator, Model
+from pyspark.ml.param.shared import *
+from bigdl.util.common import *
+from pyspark.ml.wrapper import JavaParams
+
+if sys.version >= '3':
+    long = int
+    unicode = str
+
+
+class HasBatchSize(Params):
```

Comment:
Do we need a HasMaxEpoch class too?

Reply:
added and also create a HasFeatureSize class for DLModel